CMake/External_Protobuf.cmake

```diff
@@ -24,6 +24,14 @@ else()
   set(Protobuf_PATCH_COMMAND "")
 endif()
 
+if(NOT BUILD_SHARED_LIBS)
+  set(_shared_lib_params --disable-shared
+                         --enable-static
+                         --enable-pic
+                         CFLAGS=-fPIC
+                         CXXFLAGS=-fPIC
```

Comment:
Shouldn't the C and CXX flags get appended, or is there stuff in there that will interfere?

Reply:
Sorry didn't get the question, are these flags already added by default by fletch?

Reply:
Addressed in 7255a5d